src/odemis/driver/test/xt_client_test.py

```diff
@@ -189,10 +189,13 @@ def test_set_ht_voltage(self):
 
     def test_blank_beam(self):
         """Test that the beam is unblanked after unblank beam is called, and blanked after blank is called."""
+        self.microscope.set_scan_mode("full_frame")
+        self.microscope.set_channel_state('electron1', True)
         self.scanner.blanker.value = False
         self.assertFalse(self.scanner.blanker.value)
         self.scanner.blanker.value = True
         self.assertTrue(self.scanner.blanker.value)
+        self.microscope.set_channel_state('electron1', False)
```

Comment:
No. These extra functions are not part of the Odemis API. It means that to turn off the blanker the user wouldn't be able to do it from the command line with `odemis-cli --set-attr e-beam blanker False`, and even the GUI code would be unlikely to do the right thing (as it's a generic code for all types of SEM).

I think that the proper approach is to call these functions in the VA setter. You can assume that the user is not using/touching the standard SEM software.